src/frontend/pages/game.js

```diff
@@ -34,14 +43,16 @@ export default function Home() {
         <div className={styles.actionBar}>
           <div className={styles.actionBarLeft}>
             {/* TODO: both of these values will be given by an API/socket */}
-            <h4>You Are: </h4>
-            <h4>Current Coins: </h4>
+            {/* TODO: add some JS to change styling of the color based on what color it is  */}
+            <h4 className={styles.actionBarText}>
+              You Are: <span style={{ color: 'green' }}>{thisColor} </span>
```

Comment:
is there some reason we can't use our green const, or thisColor (sorry not super familiar with style